src/quo2/components/inputs/browser_input/style.cljs

```diff
@@ -0,0 +1,24 @@
+(ns quo2.components.inputs.browser-input.style
+  (:require [quo2.components.markdown.text :as text]))
+
+(def clear-icon-container
+  {:align-items     :center
+   :height          20
+   :justify-content :center
+   :margin-left     8
+   :width           20})
+
+(def container
+  {:align-items    :center
+   :flex           1
+   :flex-direction :row})
+
+(defn input-container
+  [disabled?]
+  (assoc (text/text-style {:size   :paragraph-1
```

Comment:
hey @ulisesmac how is this implemented in text-input? also with `text/text-style` usage?

Reply:
@flexsurfer Yes, also calling this function to get the styles